addons/account/models/account_move_line.py

```diff
@@ -1676,7 +1676,7 @@ def write(self, vals):
                                 body=msg,
                                 tracking_value_ids=tracking_value_ids
                             )
-            if not self.env.context.get('skip_analytic_sync'):
+            if not self.env.context.get('skip_analytic_sync') and not self.env.context.get('posting_move'):
```

Comment:
Better to backport the [condition added in 18.4](https://github.com/odoo/odoo/blob/afac3cd16ad16ce279a87d390d7e9dca727b9b30/addons/account/models/account_move_line.py#L1786) :)

```suggestion
            if not self.env.context.get('skip_analytic_sync') and 'analytic_line_ids' in vals:
```